kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyDeployment.java

```diff
@@ -91,7 +91,7 @@ static Map<String, String> podLabels(KafkaProxy primary) {
                 .map(PodTemplateSpec::getMetadata)
                 .map(ObjectMeta::getLabels)
                 .orElse(Map.of());
-        HashMap<String, String> result = new HashMap<>(APP_KROXY);
+        Map<String, String> result = new LinkedHashMap<>(APP_KROXY);
```

Comment:
We still baseline on jdk17 don't we? Is we were on 21 we could use [`SequencedMap`](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/util/SequencedMap.html) as the LHS

Reply:
yeahp 17 I believe.